### ISSUE TITLE
imagerotate argument list is wrong

### ISSUE THREAD

**Issue by ryanguill** [foundeo/cfdocs#467]
http://cfdocs.org/imagerotate

This is a weird one, but the `angle` argument is in a different position based on how many arguments are passed.

https://helpx.adobe.com/coldfusion/cfml-reference/coldfusion-functions/functions-h-im/imagerotate.html

```
ImageRotate(name, angle [,interpolation])
ImageRotate(name, x , y , angle [,interpolation])
```

**Comment by pfreitag** [comment]
Ok, so here is what I came up with:

`imageRotate(name [, x] [, y] , angle [, interpolation])`

**Comment by pfreitag** [comment]
Fixed this in commit b331fd1
